man/cross.Rd

```diff
@@ -0,0 +1,18 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
- [ ] Re-run `devtools::document()` to delete this file, since `cross()` is now removed